### PR TITLE
Sign up: show nav links if user has existing account

### PR DESCRIPTION
**Pull request by Patrick1904**

### CODE REVIEW

File: src/components/responsive/ResponsiveContainer.js [hunk 1]
```diff
@@ -13,27 +13,33 @@ class ResponsiveContainer extends Component {
...
-    userInSignupFlow = () => {
+    showNavLinks = () => {
+        const { account, location } = this.props;
+        const accounts = account.accounts;
         const signUpRoutes = ['create', 'set-recovery', 'setup-seed-phrase'];
-        const currentBaseRoute = this.props.location.pathname.replace(/^\/([^\/]*).*$/, '$1');
-        return signUpRoutes.includes(currentBaseRoute);
+        const currentBaseRoute = location.pathname.replace(/^\/([^\/]*).*$/, '$1');
+
+        if (signUpRoutes.includes(currentBaseRoute))
```
Comment (vgrichina) [inline src/components/responsive/ResponsiveContainer.js]: nit: let's always use `{}` brackets for statements if it's not a one-liner (e.g. `if (someThing) return anotherThing;`). Otherwise it can be confusing at times.

File: src/components/responsive/ResponsiveContainer.js [hunk 2]
```diff
@@ -13,27 +13,33 @@ class ResponsiveContainer extends Component {
...
-        const { account } = this.props;
-
-        const showNavbarLinks = !this.userInSignupFlow() && account.accountId;
 
         return (
             <>
                 <DesktopView
                     {...this.props}
-                    showNavbarLinks={showNavbarLinks}
+                    showNavbarLinks={this.showNavLinks()}
```
[the remark this answers is not in the document]
Comment (Patrick1904) [reply src/components/responsive/ResponsiveContainer.js]: Makes sense will add to todo list :)

File: src/components/responsive/ResponsiveContainer.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { Responsive } from 'semantic-ui-react';
 import { connect } from 'react-redux';
 import MobileView from './MobileView';
 import DesktopView from './DesktopView';
+import { booleanLiteral } from '@babel/types';
```
Comment (vgrichina) [inline src/components/responsive/ResponsiveContainer.js]: why is this needed?

Comment (Patrick1904) [reply src/components/responsive/ResponsiveContainer.js]: Great question! Totally missed that it was automatically added. Will fix.
